tools/sci/messages_export.py

```diff
@@ -63,7 +65,7 @@ def messages_export(gamedir, csvdir):
         with open(filename, "rb") as f:
             lob = list(f.read())
 
-        assert bytes(lob[:2]) == SIERRA_MESSAGE_HEADER
+        assert bytes(lob[:2]) in {SIERRA_MESSAGE_HEADER, TRADUSCI_MESSAGE_HEADER}, bytes(lob[:2])
```

Comment:
I don't understand the ending `, bytes(lob[:2])`.
What does it do?
What will happen if we remove it?

Or maybe it's a typo?

Reply:
This value after the comma is the assertion error message.
so, when failing here, instead of just knowing the value is not expected,
we actually get to see what was the actual given value is.

```python
>>> assert False
Traceback (most recent call last):   
  File "<stdin>", line 1, in <module>
AssertionError
>>> assert False, 'something'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AssertionError: something
```

Reply:
Great!